pkg/virtctl/vnc/vnc.go

```diff
@@ -84,37 +84,51 @@ type VNC struct {
 	clientConfig clientcmd.ClientConfig
 }
 
-func (o *VNC) Run(cmd *cobra.Command, args []string) error {
-	namespace, _, err := o.clientConfig.Namespace()
+func (o *VNC) createLocalProxyForVmiWebsocketConn(vmiName string) (*net.TCPListener, error) {
+	lnAddr, err := net.ResolveTCPAddr("tcp", fmt.Sprintf("127.0.0.1:%d", customPort))
 	if err != nil {
-		return err
+		return nil, fmt.Errorf("Can't resolve the address: %s", err.Error())
```

Comment:
Isn't `%v` with `err` the common way to embed the original error text?

Reply:
These days it's `%w` which will wrap the error: https://blog.golang.org/go1.13-errors

Reply:
%w is to be used only if one is interested to expose it as an api of the package. I do not think it is a requirement here, especially when the caller has no way to use it as such.

Reply:
Right, then it should be `%v`.